### ISSUE TITLE
Add episode status to episode view in Metabase

### ISSUE THREAD

**Issue by mulundapm** [bcc-code/bcc-media-platform#441]
Status of whether episode is in draft, published, archived or unlisted is not shown in metabase

https://metabase.bcc.media/question#eyJkYXRhc2V0X3F1ZXJ5Ijp7ImRhdGFiYXNlIjo5LCJxdWVyeSI6eyJzb3VyY2UtdGFibGUiOjM5NDN9LCJ0eXBlIjoicXVlcnkifSwiZGlzcGxheSI6InRhYmxlIiwidmlzdWFsaXphdGlvbl9zZXR0aW5ncyI6e319

**Comment by mulundapm** [comment]
@KillerX Currently theres only 1 episode with a status, rest would come soon?

**Comment by KillerX** [comment]
Yeah, it's unclear why the updates are failing. I didn't see anything in the logs last time I checked but there was no concentrated effort either.
